packages/components/src/placeholder/index.tsx

```diff
@@ -72,10 +74,17 @@ export function Placeholder(
 		modifierClassNames,
 		withIllustration ? 'has-illustration' : null
 	);
+
 	const fieldsetClasses = classnames( 'components-placeholder__fieldset', {
 		'is-column-layout': isColumnLayout,
 	} );
 
+	useEffect( () => {
+		if ( instructions ) {
+			speak( instructions );
+		}
+	}, [ instructions ] );
```

Comment:
On page load, all block with placeholders with instructions are now all "speaking" at once. Is this by intention?

Reply:
@ellatrix No, it needs to be fixed.